python/lsst/ap/association/packageAlerts.py

```diff
@@ -165,6 +177,80 @@ def createDiaSourceBBox(self, bboxSize):
             bbox = geom.Extent2I(bboxSize, bboxSize)
         return bbox
 
+    def createCcdDataCutout(self, cutout, sphPoint, photoCalib):
```

Comment:
I feel `sphPoint` describes the data type more than what the parameter is. Maybe something like `centerPoint` instead?

Reply:
Changed to skyCenter